packages/xo-web/src/xo-app/vm/tab-disks.js

```diff
@@ -370,6 +394,11 @@ class NewDisk extends Component {
             </ActionButton>
           </span>
         </fieldset>
+        {!this._isSrOnSameHost() && (
+          <div>
+            <WarningIllegalSr requiredHost={requiredHost} />
+          </div>
```

Comment:
If `WarningIllegalSr` is meant to be a block element, make it a block element itself.